pennylane_lightning/core/src/simulators/lightning_kokkos/gates/tests/Test_StateVectorKokkos_Param.cpp

```diff
@@ -410,16 +622,34 @@ TEMPLATE_TEST_CASE("StateVectorKokkosManaged::applyControlledGlobalPhase",
                                          {0.0, 1.}, {1.0, 0.}, {1.0, 0.},
                                          {1.0, 0.}, {1.0, 0.}};
 
-    auto sv_data = createRandomStateVectorData<TestType>(re, num_qubits);
-    StateVectorKokkos<TestType> kokkos_sv(
-        reinterpret_cast<ComplexT *>(sv_data.data()), sv_data.size());
-    kokkos_sv.applyOperation("C(GlobalPhase)", {index}, inverse, {}, phase);
-    auto result_sv = kokkos_sv.getDataVector();
-    for (std::size_t j = 0; j < exp2(num_qubits); j++) {
-        ComplexT tmp = (inverse) ? conj(phase[j]) : phase[j];
-        tmp *= ComplexT(sv_data[j]);
-        CHECK((real(result_sv[j])) == Approx(real(tmp)));
-        CHECK((imag(result_sv[j])) == Approx(imag(tmp)));
+    SECTION("C(GlobalPhase)") {
+        auto sv_data = createRandomStateVectorData<TestType>(re, num_qubits);
+        StateVectorKokkos<TestType> kokkos_sv(
+            reinterpret_cast<ComplexT *>(sv_data.data()), sv_data.size());
+        kokkos_sv.applyOperation("C(GlobalPhase)", {index}, inverse, {}, phase);
+        auto result_sv = kokkos_sv.getDataVector();
+        for (std::size_t j = 0; j < exp2(num_qubits); j++) {
+            ComplexT tmp = (inverse) ? conj(phase[j]) : phase[j];
+            tmp *= ComplexT(sv_data[j]);
+            CHECK((real(result_sv[j])) == Approx(real(tmp)));
+            CHECK((imag(result_sv[j])) == Approx(imag(tmp)));
+        }
+    }
+
+    SECTION("Controlled GlobalPhase") {
+        const TestType pi2 = 1.5707963267948966;
+        auto sv_data = createRandomStateVectorData<TestType>(re, num_qubits);
+        StateVectorKokkos<TestType> kokkos_sv(
+            reinterpret_cast<ComplexT *>(sv_data.data()), sv_data.size());
+        kokkos_sv.applyOperation("GlobalPhase", {0, 1}, {0, 1}, {2}, inverse,
```

Comment:
Do you have any plan to test these named gates with multiple controlled wires and values?

Reply:
I will add some tests for multiple control wires/values. More tests will be added when controlled matrix is supported

Reply:
tests added.